src/main/java/frc/robot/subsystems/drivetrain/commands/TankDrive.java

```diff
@@ -0,0 +1,33 @@
+package frc.robot.subsystems.drivetrain.commands;
+
+import edu.wpi.first.wpilibj.GenericHID;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.OI;
+import frc.robot.RobotContainer;
+import frc.robot.subsystems.drivetrain.SwerveDrive;
+import frc.robot.subsystems.drivetrain.SwerveModule;
+
+public class TankDrive extends CommandBase {
+    private SwerveDrive swerveDrive;
+
+    public TankDrive(SwerveDrive swerveDrive) {
+        this.swerveDrive = swerveDrive;
+        addRequirements(swerveDrive);
+    }
+
+    @Override
+    public void initialize() {
+        for (SwerveModule swerveModule : swerveDrive.swerveModules) {
+            swerveModule.setAngle(0);
+        }
+    }
+
+    @Override
+    public void execute() {
+        swerveDrive.getModule(0).setSpeed(OI.xbox.getY(GenericHID.Hand.kRight));
+        swerveDrive.getModule(2).setSpeed(OI.xbox.getY(GenericHID.Hand.kRight));
+
+        swerveDrive.getModule(1).setSpeed(OI.xbox.getY(GenericHID.Hand.kLeft));
+        swerveDrive.getModule(3).setSpeed(OI.xbox.getY(GenericHID.Hand.kLeft));
```

Comment:
Extract `OI.xbox.getY(GenericHID.Hand.kRight)` and `OI.xbox.getY(GenericHID.Hand.kLeft)` to variables, to reduce function calls.